counsel.el

```diff
@@ -2619,16 +2619,19 @@ string - the full shell command to run."
 
 (defun counsel-locate-cmd-default (input)
   "Return a `locate' shell command based on regexp INPUT."
-  (counsel-require-program "locate")
-  (format "locate -i --regex %s"
+  (counsel-require-program locate-command)
```

Comment:
This needs a `(defvar locate-command)` at top-level so that the byte-compiler knows it's a dynamic variable that will be bound at runtime.